ros2_batch_job/__main__.py

```diff
@@ -79,6 +79,10 @@
         'cryptography',
         'lxml',
     ]
+if sys.platform == 'win32' and not args.cmake_build_type == 'Debug':
+    pip_dependencies += [
+        'opencv-python',
```

Comment:
Will opencv-python from pypi work with our opencv binary?

Reply:
A follow up question would be why doesn't our opencv binary include opencv-python?

Reply:
That could potentially fix the debug builds as well.

Reply:
> Will opencv-python from pypi work with our opencv binary?

Yes and no.  opencv-python is its own compiled version of the library, and doesn't use our version.  Because of this, there is a potential of version drift between them (we currently package OpenCV 3.4, while opencv-python from pip is currently 4.10).

> A follow up question would be why doesn't our opencv binary include opencv-python?

Yeah, maybe that is the right way to go.  I guess we'd have to figure out how to make that compile happen, and then we'd also have to figure out how to tell users to unpack properly to `C:\python37` and/or setup `PYTHONPATH`.  @mjcarroll , up for a challenge? :).

> That could potentially fix the debug builds as well.

Only if you can include both the debug and release `.pyd` files in the same tarball.  Can you?

Reply:
> Only if you can include both the debug and release `.pyd` files in the same tarball. Can you?

I believe there's some dark magic in the CI scripts to uninstall and reinstall python extensions on Windows so the correct Debug or Release library is used.